### PR TITLE
PKGBUILD file in development

### PR DESCRIPTION
**Pull request by jasursadikov**

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Maintainer: Jasur Sadikov <jasur@sadikoff.com>
+pkgname=mud
+pkgver=v1.0.2
```
Comment (emitrackman) [inline PKGBUILD:3]: most packages I see in repos are not using v in front

Comment (jasursadikov) [reply PKGBUILD:3]: We already have tags with `v` prefix :(

File: PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+pkgrel=1
+pkgdesc="Multi repository git utility. Manage multiple git-repositories simultaneously."
+arch=('any')
+url="https://github.com/jasursadikov/mud"
+license=('MIT')
+depends=('python' 'python-prettytable')
+makedepends=('python-build' 'python-installer' 'git')
+source=("${_pkgname}::git+https://github.com/jasursadikov/mud.git")
```
Comment (emitrackman) [inline PKGBUILD:11]: if you use git you package should be called mud-git by convention in aur

File: PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+md5sums=('SKIP')
+
+pkgver() {
+    cd "$srcdir/$_pkgname"
+    echo $(grep '^version =' pyproject.toml | cut -d '"' -f2)
```
Comment (emitrackman) [inline PKGBUILD:16]: just noticed it says 1.0.1 in your pyproject.toml, but pkgver is already 1.0.2

Comment (jasursadikov) [reply PKGBUILD:16]: That version is not pushed to pipy, this is the reason :)
But the packaging is not working propertly, I am failing to build :(

**Pull request merged**